client/src/app/pages/applications/components/application-detail-drawer/application-detail-drawer.tsx

```diff
@@ -182,6 +184,18 @@ const TabDetailsContent: React.FC<{
         .filter((fullArchetype) => fullArchetype?.review)
         .filter(Boolean);
 
+  const issueReportsQuery = useFetchIssueReports(application.id);
+  const {
+    result: { data, total: totalReportCount },
+  } = issueReportsQuery;
+  const currentPageReports = data;
+  const [minor, critical] = fork(currentPageReports, (u) => u.effort <= 1).map(
```

Comment:
@sjd78 @dymurray what makes an issue a critical one please?
Here is a screenshot of analysis with one **mandatory** issue which is summarized as a minor issue (effort =1)
![Screenshot_20250209_210537](https://github.com/user-attachments/assets/756adb8f-a856-44f1-ac8c-fe8e8bb26d61)